packages/cli/src/__tests__/incomplete-schemas.test.ts

```diff
@@ -30,71 +30,6 @@ DROP TABLE IF EXISTS 'test-dbexecute';
 CREATE TABLE 'test-dbexecute' ("id" INTEGER PRIMARY KEY);
 DROP TABLE 'test-dbexecute';`
 
-type Source = 'get-config' | 'validate'
-const sourceToContext = {
-  'get-config': 'getConfig',
-  validate: 'validate',
-}
-
-function urlIsMissingValidationError(source: Source) {
-  return `
-  Prisma schema validation - (${source} wasm)
-  Error code: P1012
-  error: Argument "url" is missing in data source block "db".
-    -->  schema.prisma:3
-     | 
-   2 | 
-   3 | datasource db {
-   4 |   provider = "postgresql"
-   5 | }
-     | 
-  
-  Validation Error Count: 1
-  [Context: ${sourceToContext[source]}]
-  
-  Prisma CLI Version : 0.0.0
-  `
-}
-
-const envVarNotFoundValidationError = `
-  Prisma schema validation - (get-config wasm)
-  Error code: P1012
-  error: Environment variable not found: SOME_UNDEFINED_DB.
-    -->  schema.prisma:5
-     | 
-   4 |   provider = "postgresql"
-   5 |   url      = env("SOME_UNDEFINED_DB")
-     | 
-
-  Validation Error Count: 1
-  [Context: getConfig]
-
-  Prisma CLI Version : 0.0.0
-  `
-
-const urlMustStartWithProtocolValidationError = `
- Prisma schema validation - (get-config wasm)
- Error code: P1012
- error: Error validating datasource \`db\`: the URL must start with the protocol \`postgresql://\` or \`postgres://\`.
-   -->  schema.prisma:5
-    | 
-  4 |   provider = "postgresql"
-  5 |   url      = env("SOME_DEFINED_INVALID_URL")
-    | 
- 
- Validation Error Count: 1
- [Context: getConfig]
- 
- Prisma CLI Version : 0.0.0
-   `
-
-const aDatasourceBlockIsMissingError = `A datasource block is missing in the Prisma schema file.`
-const thereIsNoDatasourceError = `
-There is no datasource in the schema.
-
-
-`
-
```

Comment:
Why did you do these changes?

Reply:
Pulling inline snapshot into separate variable does not make sense - it can't be updated this way.

Reply:
I think it was made like this for a reason though if I remember right, I think it was to make sure the snapshots are the same, this breaks this, though it's not a big deal I guess.